main/main.cpp

```diff
@@ -434,6 +434,8 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --disable-render-loop             Disable render loop so rendering only occurs when called explicitly from script.\n");
 	OS::get_singleton()->print("  --disable-crash-handler           Disable crash handler when supported by the platform code.\n");
 	OS::get_singleton()->print("  --fixed-fps <fps>                 Force a fixed number of frames per second. This setting disables real-time synchronization.\n");
+	OS::get_singleton()->print("  --enable-delta-smoothing          When vsync is enabled, enabled frame delta smoothing.\n");
+	OS::get_singleton()->print("  --disable-delta-smoothing         Disable frame delta smoothing.\n");
```

Comment:
Let me suggest collapsing these two into something like this:
`--delta-smoothing <enable/disable>    Force frame delta smoothing to be enabled or disabled.`